config/puma.rb

```diff
@@ -21,7 +21,7 @@
 # Workers do not work on JRuby or Windows (both of which do not support
 # processes).
 #
-workers ENV.fetch("WEB_CONCURRENCY") { 2 }
+# workers ENV.fetch("WEB_CONCURRENCY") { 2 }
```

Comment:
Everything else looks good. Just remove this change. Thanks :+1:

Reply:
Done. Thank You